pkg/app/pipedv1/plugin/kubernetes/deployment/plugin.go

```diff
@@ -178,6 +179,55 @@ func (p *Plugin) executeK8sSyncStage(ctx context.Context, input *sdk.ExecuteStag
 		return sdk.StageStatusSuccess
 	}
 
+	// Wait for all applied manifests to be stable.
+	// In theory, we don't need to wait for them to be stable before going to the next step
+	// but waiting for a while reduces the number of Kubernetes changes in a short time.
+	lp.Info("Waiting for the applied manifests to be stable")
+	select {
+	case <-time.After(15 * time.Second):
+		break
+	case <-ctx.Done():
+		break
+	}
+
+	lp.Info("Start finding all running resources but no longer defined in Git")
+
+	namespacedLiveResources, clusterScopedLiveResources, err := provider.GetLiveResources(ctx, kubectl, deployTargetConfig.KubeConfigPath, input.Request.Deployment.ApplicationID)
+	if err != nil {
+		lp.Errorf("Failed while getting live resources (%v)", err)
+		return sdk.StageStatusFailure
+	}
+
+	if len(namespacedLiveResources)+len(clusterScopedLiveResources) == 0 {
+		lp.Info("There is no data about live resource so no resource will be removed")
+		return sdk.StageStatusFailure
```

Comment:
This case should be success
```suggestion
		return sdk.StageStatusSuccess
```

Reply:
@Warashi 
oops, thanks! fixed on a9c1bd832e40c81c0addf85c88ec5bef97698005